sass/layouts/slider.scss

```diff
@@ -0,0 +1,67 @@
+.slider__background {
+  background: $color-slider-bg;
+  border-bottom: 6px solid $color-slider-shadow;
+  position: relative;
+  top: 66px;
+  @include media-768 {
+    top: 47px;
+  }
+}
+
+.slider {
+  display: flex;
+  align-items: baseline;
+  max-width: 1021px;
+  margin: auto;
+  position: relative;
+  overflow: hidden;
+}
+
+.slider-line {
+  width: 2050px;
+  display: flex;
+  position: relative;
+  left: 0;
+  transition: all ease 1s;
+}
+
+#home {
+  position: relative;
+  width: 10px;
+  height: 10px;
+}
+
+.slider-controls {
+  position: absolute;
+  margin: 30% auto;
+  padding: 0 30px;
+  width: 100%;
+  height: auto;
+  z-index: 3;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  @include media-768 {
+    padding: 0 10px;
+  }
+}
+
+.arrow-left,
+.arrow-right {
+  @include media-768 {
+    width: 8px;
+    height: 13.5px;
+  }
+}
+
+.arrow-left:hover {
+  cursor: pointer;
+  transform: scale(1.5);
+  @extend %transition;
+}
+
+.arrow-right:hover {
+  cursor: pointer;
+  transform: scale(1.5);
+  @extend %transition;
+}
```

Comment:
You repeat the same code for both arrows. Why not join it?

Reply:
DONE!